### ISSUE TITLE
Typing starts search

### ISSUE THREAD

**Issue by kcmannem** [zaru/menuffy#1]
It would be improved if typing started to invoke search instead of highlighting menu items that match the letters. I will almost never go through the dropdown to do what I want.

**Comment by dutzi** [comment]
I agree. I'll add that while typing should invoke search, hitting the up/down arrows should still navigate the menu

**Comment by zaru** [comment]
I think so, too. However, it's not easy.

I'm going to challenge you to make this work. I'm sure it will take some time, so I'm glad you're willing to wait.
